README.md

```diff
@@ -9,8 +9,31 @@ An event-driven image processing pipeline for developing our foundational capabi
 
 | Name    | Description                                                 |
 |---------|-------------------------------------------------------------|
+| `catalogs-aviris` | Generate a STAC Catalog for AVIRIS data |
+| `console` | Open a shell in the `dev` container |
+| `db-console` | Open a psql shell on the `database` container |
 | `infra` | Execute Terraform subcommands with remote state management. |
+| `migrate` | Run database migrations |
+| `run-activator-aviris-l2` | Run the AVIRIS L2 Activator in the development environment |
+| `server` | Start application servers, including Franklin |
+| `setup` | Run project setup after checkout |
+| `update` | Update project dependencies and rebuild containers |
 
-## STAC Catalogs
+## Application
 
-See the [catalogs README](./catalogs/README.md) for more information on catalog locations and how to build, run, and upload them.
+### Requirements
+
+- Docker
+- Docker Compose
+
+### Setup
+
+Run `./scripts/setup`.
+
+### Development
```

Comment:
I would move these subheadings above the Scripts subheading and update the table of contents.

Reply:
Agreed. 👍